spec/system/support_users/feedback_spec.rb

```diff
@@ -44,7 +44,7 @@ def testid_for(time)
       comment: "Some other feedback text",
       occupation: "Student",
       rating: "highly_satisfied",
-      email: "faketestingemail@someemail.com",
+      email: "faketestingemail12786@gmail.com",
```

Comment:
we have been using contoso.com for fake email addresses that pass the validation check - the pattern we have used is Faker::Internet.email(domain: "contoso.com")

Reply:
Ah thanks @starswan, will update!